Add recipe for evedel

daedsidog opened the pull request:
### Brief summary of what the package does

Evedel is a gptel extension for managing and processing in-buffer instructions efficiently and ergonomically. It provides functionality for defining, manipulating, and processing LLM directives and references in buffers.
 
### Direct link to the package repository

https://github.com/daedsidog/evedel

### Your association with the package

I am the author and maintainer.

### Relevant communications with the upstream package maintainer

n/a

### Checklist

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback **(well, most of it. It does not like the fact that I am using the new reader shorthands, for some reason).**
- [x] My elisp byte-compiles cleanly
- [x] I've used `M-x checkdoc` to check the package's documentation strings **(the tool seems to be broken, though, as it just hangs).**
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

riscy commented:
Thanks for this.  I see your note re: package-lint but I believe `read-symbol-shorthands` if for specifying name prefixes only, whereas you're using it to specify a name prefix *and* a non-standard separator.  I also think that a short package name like evedel should not necessarily need to be shortened to `e` - and I would recommend only using `read-symbol-shorthands` in very exceptional cases.  Beyond that, quick first pass -

⸺ `evedel.el` with byte-compile using Emacs 29.4:
```
evedel.el:1095:2: Warning: docstring wider than 80 characters
In end of data:
evedel.el:696:14: Warning: the function `markdown-mode' is not known to be defined.
```

⸺ `evedel.el` with checkdoc 0.6.2 (fix *within reason*):
```
evedel.el:588: First line is not a complete sentence
evedel.el:845: All variables and subroutines might as well have a documentation string
evedel.el:848: All variables and subroutines might as well have a documentation string
evedel.el:854: All variables and subroutines might as well have a documentation string
evedel.el:1096: Some lines are over 80 columns wide
evedel.el:1185: Argument `include-parent-tags' should appear (as INCLUDE-PARENT-TAGS) in the doc string
evedel.el:1331: All variables and subroutines might as well have a documentation string
evedel.el:1334: All variables and subroutines might as well have a documentation string
evedel.el:1429: Argument `instr' should appear (as INSTR) in the doc string
evedel.el:1921: All variables and subroutines might as well have a documentation string
evedel.el:1950: All variables and subroutines might as well have a documentation string
evedel.el:2638: All variables and subroutines might as well have a documentation string
evedel.el:2648: All variables and subroutines might as well have a documentation string
evedel.el:936: White space found at end of line
```

⸺ `evedel.el` with package-lint 20240923.1044:
```
225 issues found:
1:0: warning: Including "Emacs" in the package summary is usually redundant.
43:0: error: "e:reference-color" doesn't start with package's prefix "evedel".
43:0: error: `e:reference-color' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
47:0: error: "e:directive-color" doesn't start with package's prefix "evedel".
47:0: error: `e:directive-color' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
51:0: error: "e:directive-processing-color" doesn't start with package's prefix "evedel".
51:0: error: `e:directive-processing-color' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
55:0: error: "e:directive-success-color" doesn't start with package's prefix "evedel".
55:0: error: `e:directive-success-color' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
59:0: error: "e:directive-fail-color" doesn't start with package's prefix "evedel".
59:0: error: `e:directive-fail-color' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
63:0: error: "e:highlighted-instruction-color" doesn't start with package's prefix "evedel".
63:0: error: `e:highlighted-instruction-color' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
67:0: error: "e:instruction-bg-tint-intensity" doesn't start with package's prefix "evedel".
67:0: error: `e:instruction-bg-tint-intensity' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
71:0: error: "e:instruction-label-tint-intensity" doesn't start with package's prefix "evedel".
71:0: error: `e:instruction-label-tint-intensity' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
75:0: error: "e:highlighted-instruction-tint-intensity" doesn't start with package's prefix "evedel".
75:0: error: `e:highlighted-instruction-tint-intensity' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
79:0: error: "e:subinstruction-tint-coefficient" doesn't start with package's prefix "evedel".
79:0: error: `e:subinstruction-tint-coefficient' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
88:0: error: "e:empty-tag-query-matches-all" doesn't start with package's prefix "evedel".
88:0: error: `e:empty-tag-query-matches-all' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
96:0: error: "e:always-match-untagged-references" doesn't start with package's prefix "evedel".
96:0: error: `e:always-match-untagged-references' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
108:0: error: "e:patch-outdated-instructions" doesn't start with package's prefix "evedel".
108:0: error: `e:patch-outdated-instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
113:0: error: "e:descriptive-mode-roles" doesn't start with package's prefix "evedel".
113:0: error: `e:descriptive-mode-roles' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
125:0: error: "e::instructions" doesn't start with package's prefix "evedel".
125:0: error: `e::instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
127:0: error: "e::default-instruction-priority" doesn't start with package's prefix "evedel".
127:0: error: `e::default-instruction-priority' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
128:0: error: "e::highlighted-instruction" doesn't start with package's prefix "evedel".
128:0: error: `e::highlighted-instruction' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
129:0: error: "e::inhibit-file-restoration" doesn't start with package's prefix "evedel".
129:0: error: `e::inhibit-file-restoration' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
133:0: error: "e::id-counter" doesn't start with package's prefix "evedel".
133:0: error: `e::id-counter' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
134:0: error: "e::id-usage-map" doesn't start with package's prefix "evedel".
134:0: error: `e::id-usage-map' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
135:0: error: "e::retired-ids" doesn't start with package's prefix "evedel".
135:0: error: `e::retired-ids' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
137:0: error: "e::foreach-instruction" doesn't start with package's prefix "evedel".
137:0: error: `e::foreach-instruction' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
194:0: error: "e:version" doesn't start with package's prefix "evedel".
194:0: error: `e:version' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
207:0: error: "e:save-instructions" doesn't start with package's prefix "evedel".
207:0: error: `e:save-instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
255:0: error: "e:load-instructions" doesn't start with package's prefix "evedel".
255:0: error: `e:load-instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
306:0: error: "e:instruction-count" doesn't start with package's prefix "evedel".
306:0: error: `e:instruction-count' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
326:0: error: "e:create-reference" doesn't start with package's prefix "evedel".
326:0: error: `e:create-reference' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
340:0: error: "e:create-directive" doesn't start with package's prefix "evedel".
340:0: error: `e:create-directive' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
353:0: error: "e:link-instructions" doesn't start with package's prefix "evedel".
353:0: error: `e:link-instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
393:0: error: "e:unlink-instructions" doesn't start with package's prefix "evedel".
393:0: error: `e:unlink-instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
433:0: error: "e:cycle-instructions-at-point" doesn't start with package's prefix "evedel".
433:0: error: `e:cycle-instructions-at-point' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
461:0: error: "e:modify-directive" doesn't start with package's prefix "evedel".
461:0: error: `e:modify-directive' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
470:0: error: "e:modify-reference-commentary" doesn't start with package's prefix "evedel".
470:0: error: `e:modify-reference-commentary' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
477:0: error: "e:process-directives" doesn't start with package's prefix "evedel".
477:0: error: `e:process-directives' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
530:0: error: "e:delete-instructions" doesn't start with package's prefix "evedel".
530:0: error: `e:delete-instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
556:0: error: "e:delete-all-instructions" doesn't start with package's prefix "evedel".
556:0: error: `e:delete-all-instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
587:0: error: "e:convert-instructions" doesn't start with package's prefix "evedel".
587:0: error: `e:convert-instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
643:0: error: "e:next-instruction" doesn't start with package's prefix "evedel".
643:0: error: `e:next-instruction' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
649:0: error: "e:previous-instruction" doesn't start with package's prefix "evedel".
649:0: error: `e:previous-instruction' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
655:0: error: "e:next-reference" doesn't start with package's prefix "evedel".
655:0: error: `e:next-reference' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
661:0: error: "e:previous-reference" doesn't start with package's prefix "evedel".
661:0: error: `e:previous-reference' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
667:0: error: "e:next-directive" doesn't start with package's prefix "evedel".
667:0: error: `e:next-directive' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
673:0: error: "e:previous-directive" doesn't start with package's prefix "evedel".
673:0: error: `e:previous-directive' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
679:0: error: "e:preview-directive-prompt" doesn't start with package's prefix "evedel".
679:0: error: `e:preview-directive-prompt' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
705:0: error: "e:modify-directive-tag-query" doesn't start with package's prefix "evedel".
705:0: error: `e:modify-directive-tag-query' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
726:0: error: "e:add-tags" doesn't start with package's prefix "evedel".
726:0: error: `e:add-tags' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
742:0: error: "e:remove-tags" doesn't start with package's prefix "evedel".
742:0: error: `e:remove-tags' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
844:0: error: "e::instruction-id" doesn't start with package's prefix "evedel".
844:0: error: `e::instruction-id' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
847:0: error: "e::stashed-buffer-instructions" doesn't start with package's prefix "evedel".
847:0: error: `e::stashed-buffer-instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
853:0: error: "e::stash-buffer" doesn't start with package's prefix "evedel".
853:0: error: `e::stash-buffer' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
866:0: error: "e::read-directive-tag-query" doesn't start with package's prefix "evedel".
866:0: error: `e::read-directive-tag-query' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
939:0: error: "e::print-instruction-not-found" doesn't start with package's prefix "evedel".
939:0: error: `e::print-instruction-not-found' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
984:0: error: "e::filter-references" doesn't start with package's prefix "evedel".
984:0: error: `e::filter-references' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
998:0: error: "e::tag-query-prefix-from-infix" doesn't start with package's prefix "evedel".
998:0: error: `e::tag-query-prefix-from-infix' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1086:0: error: "e::available-tags" doesn't start with package's prefix "evedel".
1086:0: error: `e::available-tags' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1095:0: error: "e::cycle-list-around" doesn't start with package's prefix "evedel".
1095:0: error: `e::cycle-list-around' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1107:0: error: "e::cycle-instruction" doesn't start with package's prefix "evedel".
1107:0: error: `e::cycle-instruction' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1151:0: error: "e::add-tags" doesn't start with package's prefix "evedel".
1151:0: error: `e::add-tags' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1165:0: error: "e::remove-tags" doesn't start with package's prefix "evedel".
1165:0: error: `e::remove-tags' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1179:0: error: "e::inherited-tags" doesn't start with package's prefix "evedel".
1179:0: error: `e::inherited-tags' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1184:0: error: "e::reference-tags" doesn't start with package's prefix "evedel".
1184:0: error: `e::reference-tags' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1194:0: error: "e::delete-instruction-at" doesn't start with package's prefix "evedel".
1194:0: error: `e::delete-instruction-at' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1203:0: error: "e::being-processed-p" doesn't start with package's prefix "evedel".
1203:0: error: `e::being-processed-p' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1207:0: error: "e::directive-empty-p" doesn't start with package's prefix "evedel".
1207:0: error: `e::directive-empty-p' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1220:0: error: "e::create-instruction" doesn't start with package's prefix "evedel".
1220:0: error: `e::create-instruction' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1330:0: error: "e::referencep" doesn't start with package's prefix "evedel".
1330:0: error: `e::referencep' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1333:0: error: "e::directivep" doesn't start with package's prefix "evedel".
1333:0: error: `e::directivep' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1336:0: error: "e::tint" doesn't start with package's prefix "evedel".
1336:0: error: `e::tint' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1373:0: error: "e::instruction-type" doesn't start with package's prefix "evedel".
1373:0: error: `e::instruction-type' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1381:0: error: "e::create-instruction-overlay-in" doesn't start with package's prefix "evedel".
1381:0: error: `e::create-instruction-overlay-in' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1408:0: error: "e::instruction-p" doesn't start with package's prefix "evedel".
1408:0: error: `e::instruction-p' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1412:0: error: "e::parent-instruction" doesn't start with package's prefix "evedel".
1412:0: error: `e::parent-instruction' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1428:0: error: "e::bodyless-instruction-p" doesn't start with package's prefix "evedel".
1428:0: error: `e::bodyless-instruction-p' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1432:0: error: "e::subinstruction-of-p" doesn't start with package's prefix "evedel".
1432:0: error: `e::subinstruction-of-p' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1459:0: error: "e::create-reference-in" doesn't start with package's prefix "evedel".
1459:0: error: `e::create-reference-in' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1467:0: error: "e::create-directive-in" doesn't start with package's prefix "evedel".
1467:0: error: `e::create-directive-in' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1486:0: error: "e::delete-instruction" doesn't start with package's prefix "evedel".
1486:0: error: `e::delete-instruction' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1516:0: error: "e::pos-bol-p" doesn't start with package's prefix "evedel".
1516:0: error: `e::pos-bol-p' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1523:0: error: "e::fill-label-string" doesn't start with package's prefix "evedel".
1523:0: error: `e::fill-label-string' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1589:0: error: "e::instructions-congruent-p" doesn't start with package's prefix "evedel".
1589:0: error: `e::instructions-congruent-p' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1595:0: error: "e::apply-face-to-match" doesn't start with package's prefix "evedel".
1595:0: error: `e::apply-face-to-match' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1608:0: error: "e::instruction-bufferlevel-p" doesn't start with package's prefix "evedel".
1608:0: error: `e::instruction-bufferlevel-p' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1616:0: error: "e::update-instruction-overlay" doesn't start with package's prefix "evedel".
1616:0: error: `e::update-instruction-overlay' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1896:0: error: "e::restore-overlay" doesn't start with package's prefix "evedel".
1896:0: error: `e::restore-overlay' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1906:0: error: "e::file-outdated-p" doesn't start with package's prefix "evedel".
1906:0: error: `e::file-outdated-p' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1920:0: error: "e::setup-buffer-hooks" doesn't start with package's prefix "evedel".
1920:0: error: `e::setup-buffer-hooks' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
1949:0: error: "e::buffer-has-instructions-p" doesn't start with package's prefix "evedel".
1949:0: error: `e::buffer-has-instructions-p' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2019:0: error: "e::wholly-contained-instructions" doesn't start with package's prefix "evedel".
2019:0: error: `e::wholly-contained-instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2028:0: error: "e::instructions-at" doesn't start with package's prefix "evedel".
2028:0: error: `e::instructions-at' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2042:0: error: "e::instructions-in" doesn't start with package's prefix "evedel".
2042:0: error: `e::instructions-in' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2054:0: error: "e::partially-contained-instructions" doesn't start with package's prefix "evedel".
2054:0: error: `e::partially-contained-instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2067:0: error: "e::instructions" doesn't start with package's prefix "evedel".
2067:0: error: `e::instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2097:0: error: "e::toplevel-instructions" doesn't start with package's prefix "evedel".
2097:0: error: `e::toplevel-instructions' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2120:0: error: "e::directive-text" doesn't start with package's prefix "evedel".
2120:0: error: `e::directive-text' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2126:0: error: "e::commentary-text" doesn't start with package's prefix "evedel".
2126:0: error: `e::commentary-text' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2132:0: error: "e::read-directive" doesn't start with package's prefix "evedel".
2132:0: error: `e::read-directive' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2160:0: error: "e::read-commentary" doesn't start with package's prefix "evedel".
2160:0: error: `e::read-commentary' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2183:0: error: "e::descriptive-llm-mode-role" doesn't start with package's prefix "evedel".
2183:0: error: `e::descriptive-llm-mode-role' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2197:0: error: "e::directive-llm-system-message" doesn't start with package's prefix "evedel".
2197:0: error: `e::directive-llm-system-message' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2204:0: error: "e::delimiting-markdown-backticks" doesn't start with package's prefix "evedel".
2204:0: error: `e::delimiting-markdown-backticks' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2211:0: error: "e::overlay-region-info" doesn't start with package's prefix "evedel".
2211:0: error: `e::overlay-region-info' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2270:0: error: "e::toplevel-references" doesn't start with package's prefix "evedel".
2270:0: error: `e::toplevel-references' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2279:0: error: "e::multiline-string-p" doesn't start with package's prefix "evedel".
2279:0: error: `e::multiline-string-p' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2283:0: error: "e::markdown-enquote" doesn't start with package's prefix "evedel".
2283:0: error: `e::markdown-enquote' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2297:0: error: "e::directive-llm-prompt" doesn't start with package's prefix "evedel".
2297:0: error: `e::directive-llm-prompt' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2576:0: error: "e::ancestral-commentators" doesn't start with package's prefix "evedel".
2576:0: error: `e::ancestral-commentators' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2591:0: error: "e::wordwise-diff-patch-buffers" doesn't start with package's prefix "evedel".
2591:0: error: `e::wordwise-diff-patch-buffers' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2637:0: error: "e::create-id" doesn't start with package's prefix "evedel".
2637:0: error: `e::create-id' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2647:0: error: "e::retire-id" doesn't start with package's prefix "evedel".
2647:0: error: `e::retire-id' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2652:0: error: "e::reset-id-counter" doesn't start with package's prefix "evedel".
2652:0: error: `e::reset-id-counter' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2658:0: error: "e::instruction-outlinks" doesn't start with package's prefix "evedel".
2658:0: error: `e::instruction-outlinks' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
2662:0: error: "e::instruction-inlinks" doesn't start with package's prefix "evedel".
2662:0: error: `e::instruction-inlinks' contains a non-standard separator `:', use hyphens instead (see Elisp Coding Conventions).
```

⸺ `evedel.el` with [melpazoid](https://github.com/riscy/melpazoid):
```
- The `;;; Commentary` for this file is much wider than 80 characters
- evedel.el#L1348: It's safer to sharp-quote function names; use `#'`
- evedel.el#L636: It's safer to sharp-quote function names; use `#'`
- evedel.el#L737: It's safer to sharp-quote function names; use `#'`
- evedel.el#L753: It's safer to sharp-quote function names; use `#'`
- evedel.el#L1825: Could this `point`/`pos-eol` comparison use `eolp`?
- evedel.el#L2666: Loading a package should rarely add hooks
- evedel.el#L2482: `concat` may be unneeded; `insert` can take multiple arguments
- evedel.el#L2514: `concat` may be unneeded; `insert` can take multiple arguments
- evedel.el#L2527: `concat` may be unneeded; `insert` can take multiple arguments
- evedel.el#L2556: `concat` may be unneeded; `insert` can take multiple arguments
- evedel.el#L2565: `concat` may be unneeded; `insert` can take multiple arguments
```

⸺ Package and license:
- Please specify `:fetcher` before `:repo` in your recipe
- Reminder: ensure existing GitHub release v0.4.16 is up-to-date with your current code and `Package-Version`

riscy commented:
> I hope it's possible to still keep these shorthands despite your sound advice against them. I am both extremely lazy and am fond of the CL-style.

I understand your perspective, but I have to push back against this.  Occasionally people do ask to bring in conventions that they're used to from other programming languages, but there is a community style we try to adhere to.  A `read-symbol-shorthand` would be fine if you were using standard separators.

Also note that package-lint is complaining about at least one thing that's very easy to fix, i.e.

```
1:0: warning: Including "Emacs" in the package summary is usually redundant.
```

I'm not sure about what might be happening with checkdoc on your instance, but you could try it out with `emacs -Q` or on smaller files and see if the issue persists.

daedsidog commented:
> I'm not sure about what might be happening with checkdoc on your instance, but you could try it out with emacs -Q or on smaller files and see if the issue persists.

I found what the problem was and addressed all checkdoc notes except for some of the ones asking for subroutine docstrings.

> I understand your perspective, but I have to push back against this. Occasionally people do ask to bring in conventions that they're used to from other programming languages, but there is a community style we try to adhere to. A read-symbol-shorthand would be fine if you were using standard separators.

Very well, I changed the shorthands to use the standard separators. Let the record reflect, however, that I still find the CL separators to be of superior form!
